src/edit/DistortableImage.EditToolbar.js

```diff
@@ -101,18 +101,19 @@ var EditOverlayAction = LeafletToolbar.ToolbarAction.extend({
 				title: 'Export Image'
 			}
 		},
-		
+
 		addHooks: function ()
 		{
 			var editing = this._overlay.editing;
 
 			editing._toggleExport();
-			this.disable(); 
+			this.disable();
 		}
 	});
 
-L.DistortableImage.EditToolbar = LeafletToolbar.Popup.extend({
+L.DistortableImage.EditToolbar = LeafletToolbar.Control.extend({
```

Comment:
OK, so here, I think perhaps we need to have the default still be the popup. Can we set this at initialization time using an option, and do `LeafletToolbar[options.toolbarType].extend({` here? I'd be fine with making a new example that shows the "control" style toolbar but think we should leave it up to the end user and should keep the simpler "popup" style toolbar for the default demo. 

I hope that makes sense; thank you!

Reply:
@jywarren I've incorporated this change in my lastest commit, please review.